frontend/public/components/_overview.scss

```diff
@@ -26,8 +26,72 @@
 }
 
 .overview {
-  width: 100%;
+  bottom: 0;
   display: flex;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 67px;
+  @media(min-width: $screen-sm-min) {
+    left: 220px;
+    top: 100px;
+  }
+
+  .overview__middle {
+    height: 100%;
+    overflow: hidden;
+    overflow-y: auto;
+    width: 100%;
+    -webkit-overflow-scrolling: touch;
+
+    .overview__middle-section {
+      flex-grow: 1;
+    }
+  }
+
+  .overview__sidebar--right {
```

Comment:
Since there isn't an `.overview__sidebar`, the `--right` modifier is not necessary.